packages/web-twig/src/Resources/components/Pagination/PaginationLinkPrevious.twig

```diff
@@ -0,0 +1,22 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _href = props.href -%}
+{%- set _label = props.label | default('Previous') -%}
+{%- set _target = props.target | default(null) -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+
+{% embed "@spirit/buttonLink.twig" with { props: {
```

Comment:
You cannot use TwigX syntax here?

Reply:
nope, I am not able to get there mainProps.